integration-tests/src/test/java/tachyon/master/file/FileSystemMasterIntegrationTest.java

```diff
@@ -279,6 +280,7 @@ public final void before() throws Exception {
     mFsMaster =
         mLocalTachyonClusterResource.get().getMaster().getInternalMaster().getFileSystemMaster();
     mMasterTachyonConf = mLocalTachyonClusterResource.get().getMasterTachyonConf();
+    TTLBucket.setTTlIntervalMs(mMasterTachyonConf.getLong(Constants.MASTER_TTLCHECKER_INTERVAL_MS));
```

Comment:
It could theoretically, but that would reveal a bug in the other tests where they test TTL but don't set the TTL interval. https://tachyon.atlassian.net/browse/TACHYON-1382 will address this by causing every test to reset the TTL interval to correspond with whatever TachyonConf they set up.